src/tokenizer.py

```diff
@@ -9,15 +9,15 @@
 from enum import Enum
 from pathlib import Path
 
-import src.scripts.preprocessing.kutuzov.rus_preprocessing_udpipe as kutuzov
+import scripts.preprocessing.kutuzov.rus_preprocessing_udpipe as kutuzov
 
 # Set up the configuration
 path_current_directory = os.path.dirname(__file__)
 path_config_file = os.path.join(path_current_directory, '../',
                                 'config.ini')
 config = configparser.ConfigParser()
 config.read(path_config_file)
-models_dir = config['PATHS']['models_dir']
+treetagger_dir = config['PATHS']['treetagger_dir']
```

Comment:
models_dir includes not just treetagger, but other tokenization models (udpipe), and final trained models (bert, w2v, etc.). I think it's better to keep it as models_dir or perhaps create two dirs: 'output_models_dir' and 'tokenizer/lemmatizer_dir' or something along those lines.

Reply:
I would go for the second solution, since the usage of output models and lemmatizers is distinct.